supply/lib/supply/setup.rb

```diff
@@ -40,34 +39,58 @@ def store_metadata(listing)
       end
     end
 
-    def download_images(listing)
+    def download_images(listing, image_format = "png", remove_alpha = "false")
       UI.message("🖼️  Downloading images (#{listing.language})")
 
-      # We cannot download existing screenshots as they are compressed
-      # But we can at least download the images
       require 'net/http'
 
-      IMAGES_TYPES.each do |image_type|
-        if ['featureGraphic'].include?(image_type)
-          # we don't get all files in full resolution :(
-          UI.message("📵  Due to a limitation of the Google Play API, there is no way for `supply` to download your existing feature graphic. Please copy your feature graphic to `metadata/android/#{listing.language}/images/featureGraphic.png`")
-          next
-        end
+      allowed_imagetypes = [Supply::IMAGES_TYPES, Supply::SCREENSHOT_TYPES].flatten
 
+      allowed_imagetypes.each do |image_type|
         begin
           UI.message("Downloading `#{image_type}` for #{listing.language}...")
 
-          url = client.fetch_images(image_type: image_type, language: listing.language).last
-          next unless url
+          urls = client.fetch_images(image_type: image_type, language: listing.language)
+          next unless urls
+
+          if urls.length > 0
+            image_counter = 1
```

Comment:
Why doesn't this start at `0`? 🤔

Reply:
`image_counter` is used to prefix the final images. Please see this screenshot - https://user-images.githubusercontent.com/18679225/53509689-51930c00-3a8a-11e9-8b81-4d4a7c3bbe11.png. Let me know if you'd like to have a different naming convention, please!

Reply:
Can we maybe just add a comment above this line on why this number is used?